torch/_dynamo/trace_rules.py

```diff
@@ -306,6 +306,7 @@
     "torch.jit._unwrap_optional": UserFunctionVariable,
     "torch.backends.mha.get_fastpath_enabled": UserFunctionVariable,
     "torch._dynamo.mark_static": UserFunctionVariable,
+    "torch._dynamo.mark_traceable": UserFunctionVariable,
```

Comment:
This, along with changes in `UserFunctionVariable.call_function`, enables using `mark_traceable` _inside_ a `torch.compile` region, also see `test_mark_traceable_inside_compiled_function`.